src/components/carousel/Carousel.jsx

```diff
@@ -4,13 +4,12 @@ import { Link } from "react-router-dom";
 import "./carousel.css";
 const Carousel = () => {
   const [categories, setCategories] = useState([]);
-  console.log("hsjbs");
   const fetchCategories = async () => {
     try {
       const response = await axios.get(`/api/categories`);
       setCategories(response.data.categories);
     } catch (error) {
-      console.log(error);
+      alert(error);
```

Comment:
You can use the same Toast component used in other places in your code.

Reply:
sure, i will do it later.